.slim-lint.yml

```diff
@@ -16,4 +16,4 @@ linters:
     exclude:
       - 'app/views/voice/otp/show.xml.slim'
   RuboCop:
-    enabled: true
+    enabled: false
```

Comment:
I disabled this because it causes lots of issues because of how slim compiles to ERB. There's no real way to tune it and slim is on the way out so it seemed ok to disable.